config/routes.rb

```diff
@@ -1,5 +1,28 @@
 # rubocop:disable BlockLength
 Rails.application.routes.draw do
+  scope '/' do
```

Comment:
There is already an `:api` namespace (at the bottom). Was there any reason not to put it there?

If possible it would be nice to have it in there instead of creating a duplicate namespace.